### DISCUSSION TITLE
How do I include a space in a snippet?

### DISCUSSION THREAD

**Discussion by BOBONA** [artisticat1/obsidian-latex-suite#222] [Q&A] [answered]
Hi. This might be a basic question but I'm having a hard time figuring it out. How do I include a space after a snippet? I have a snippet that replaces oo with //circ. But I have to press space afterwards if I want it to format correctly. Nothing changes when I add the space to the replacement.

Edit: it's actually a setting in the plugin, "Advanced editor settings > Remove trailing whitespaces in inline math"

**Comment by JaimeVicent** [comment]
Try this:
`{trigger: "oo", replacement: "\\circ $0", options: "mA"}`
It should work. Remember to add a tab!!

**Reply by BOBONA** [reply]
But there's no way to include a space automatically? I could do \\circ{} to get the result I want but that seems like an ugly solution.

**Reply by JaimeVicent** [reply]
I'm not sure about what you're talking. I simply deduced that "$0" meant where you wanted your cursor to focus, so it works.

**Reply by BOBONA** [reply]
Maybe this is an actual glitch then? On my machine the space is ignored, and I have to manually type a space every time.

**Reply by JaimeVicent** [reply]
Didn't it work adding the snippet `{trigger: "oo", replacement: "\\circ $0", options: "mA"}` to your snippet list? It is working fine for me. Make sure it says "Saved" and a green tick after you add it.

![image](https://github.com/artisticat1/obsidian-latex-suite/assets/116151288/54db4bdf-26b8-4afc-ae2f-2c8e072c3b9c)

**Reply by BOBONA** [reply]
No, it does not work. This is what I get after typing "oo". 
<img width="65" alt="image" src="https://github.com/artisticat1/obsidian-latex-suite/assets/25057945/fa5065b5-cb04-40c3-9c20-ac0fa339ac0f">

**Reply by JaimeVicent** [reply]
Hmm 🤷‍♂️. It must be a glitch. Check that you have latex-suite uploaded

**Comment by artisticat1** [comment, marked as answer]
Try disabling the setting "Advanced editor settings > Remove trailing whitespaces in inline math". Apologies for the confusion!
